PubNubSwiftChatSDK/PubNubSwiftChatSDK.docc/Message.md

```diff
@@ -15,40 +14,52 @@
 
 ### Receiving Updates
 
+- ``streamUpdates()``
 - ``streamUpdates(completion:)``
+- ``streamUpdatesOn(messages:)``
 - ``streamUpdatesOn(messages:callback:)``
```

Comment:
There is` messages:callback:`
Shouldn't be `messages: completion:` ?

Reply:
Here's the rule:

Swift's standard naming convention for completion arguments is `completion:` or  `completionHandler:` and this applies to one-shot asynchronous calls that signal a result once and then finish.

In your example, `callback:` could be invoked more than once so the rule I mentioned doesn't apply here. Therefore, I used the same argument name as it is in KMP to stay aligned